fig/backends/chronicle/README.md

```diff
@@ -30,6 +32,7 @@ backends=CHRONICLE
        -e FALCON_CLIENT_ID="$FALCON_CLIENT_ID" \
        -e FALCON_CLIENT_SECRET="$FALCON_CLIENT_SECRET" \
        -e GOOGLE_SECURITY_KEY="$GOOGLE_SECURITY_KEY" \
+       -e CHRONICLE_REGION="$CHRONICLE_REGION" \
```

Comment:
Where is the documentation on what acceptable values for `CHRONICLE_REGION` are?